WordPress/src/main/java/org/wordpress/android/util/NotificationsUtilsWrapper.kt

```diff
@@ -0,0 +1,10 @@
+package org.wordpress.android.util
+
+import androidx.core.app.NotificationManagerCompat
+import javax.inject.Inject
+
+class NotificationsUtilsWrapper @Inject constructor() {
```

Comment:
Suggestion: Instead of adding a new class just for clearing the notifications, can we add this function to the `NotificationsUtils` class?.

Reply:
Resolved this by adding the function to the NotificationUtils class , thank you for the suggestion